.github/workflows/ci.yml

```
@@ -1,66 +1,209 @@
+          prerelease: false
+
+      - name: Run post-deployment tests
         run: |
-          if [ ! -d "dist" ]; then
-            echo "Build failed: dist directory not found"
-            exit 1
-          fi
-          echo "Build successful - dist directory exists"
+          npm run test:deploy
+        env:
+          DEPLOYMENT_URL: ${{ steps.deploy.outputs.deployment-url }}
```

actionlint (1.7.9)

property "deploy" is not defined in object type {}

(expression)